src/Converter/Feature2Mesh.js

```diff
@@ -9,8 +9,54 @@ import OrientationUtils from 'Utils/OrientationUtils';
 import Coordinates from 'Core/Geographic/Coordinates';
 
 const coord = new Coordinates('EPSG:4326', 0, 0, 0);
+
+class FeatureContext {
+    #worldCoord = new Coordinates('EPSG:4326', 0, 0, 0);
+    #localCoordinates = new Coordinates('EPSG:4326', 0, 0, 0);
+    #geometry = {};
+
+    constructor() {
+        this.globals = {};
+    }
+
+    setGeometry(g) {
+        this.#geometry = g;
+    }
+
+    setCollection(c) {
+        this.collection = c;
+        this.#localCoordinates.setCrs(c.crs);
+    }
+
+    setLocalCoordinatesFromArray(vertices, offset) {
+        this.#worldCoord.isLocal = true;
+        return this.#localCoordinates.setFromArray(vertices, offset);
+    }
+
+    properties() {
+        return this.#geometry.properties;
+    }
+
+    get coordinates() {
+        if (this.#worldCoord.isLocal) {
+            this.#worldCoord.isLocal = false;
+            this.#worldCoord.copy(this.#localCoordinates).applyMatrix4(this.collection.matrixWorld);
```

Comment:
you can use a private boolean property (`#worldCoordAvailable` for instance) instead of worldCoord.isLocal

Reply:
Done.